src/js/base/api.js

```diff
@@ -760,6 +828,52 @@ ripe.Ripe.prototype._getPricesOptions = function(options = {}) {
     return options;
 };
 
+/**
+ * @ignore
+ */
+ripe.Ripe.prototype._getVideoOptions = function(options = {}) {
+    options = this._getQueryOptions(options);
+
+    const params = options.params || {};
+    options.params = params;
+
+    const name = options.name === undefined ? undefined : options.name;
+    if (name !== undefined && name !== null) {
+        options.params.name = name;
+    }
+
+    const url = `${this.url}video`;
+    options = Object.assign(options, {
+        url: url,
+        method: "GET"
+    });
+
+    return options;
+};
+
+/**
+ * @ignore
+ */

```

Comment:
```suggestion
/**
 * Builds the video thumbnail options.
 *
 * @param {Object} A set of options to configure the request.
 * @returns {Object} The options for the video thumbnail request.
 */
```

Reply:
used ignore since other similar functions do that as well, don't know the reason
I don't think documentation of the function adds to the overall information of the function, since it's supposed to be a private method, but I'll add the comment